placement/types.go

```diff
@@ -431,6 +431,9 @@ type Storage interface {
 
 	// Proto returns the placement proto.
 	Proto() (proto.Message, int, error)
+
+	// History returns the placement of a specific version.
+	History(version int) (Placement, error)
```

Comment:
Do we need a way to get the past version numbers themselves? Or are versions always incremented?

Reply:
versions on the placement are always incremented, when you get the latest placement you will see the current version, then you can refer the previous versions you want to look up